src/dynatrace/opentelemetry/metrics/export/serializer.py

```diff
@@ -68,12 +68,26 @@ class DynatraceMetricsSerializer:
     def __init__(
         self,
         prefix: Optional[str],
-        tags: Optional[Mapping],
+        dimensions: Optional[Mapping],
     ):
         self._prefix = prefix
-        self._tags = tags or {}
         self._is_delta_export = None
 
+        self._dimensions = self._normalize_dimensions(dimensions)
+
+    @classmethod
+    def _normalize_dimensions(cls, dimensions):
+        dim_dict = {}
+        if dimensions:
+            # normalize the dimensions once, so it doesnt have to be repeated
+            # for every serialisation
+            for k, v in dimensions.items():
+                key = cls._normalize_dimension_key(k)
+                if key:
+                    dim_dict[key] = cls._normalize_dimension_value(v)
```

Comment:
Could this return None?

Reply:
it can return an empty dimension key, which would also evaluate to `False` and will lead to invalid dimension keys not being added.